common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/ButtonConstants.kt

```diff
@@ -16,3 +17,26 @@ class ButtonConstants(theme: CurrentTheme) {
     val cornerRadius = theme.cornerRadiusScale.buttonRadius
     val defaultTruncate = true
 }
+
+class NewButtonConstants(theme: CurrentTheme) {
```

Comment:
Looking good! Presumably we won't be keeping around the "old" Button?

Reply:
Yes NewButton will replace Button

Reply:
👌 shall we make the change with this same PR? If so, we can remove the old implementation and keep the one, new `ButtonConstants`.